wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsnext/model/ChePlugin.java

```diff
@@ -96,37 +38,20 @@ public boolean equals(Object o) {
     if (!(o instanceof ChePlugin)) {
       return false;
     }
-    ChePlugin chePlugin = (ChePlugin) o;
-    return Objects.equals(getName(), chePlugin.getName())
-        && Objects.equals(getId(), chePlugin.getId())
-        && Objects.equals(getVersion(), chePlugin.getVersion())
-        && Objects.equals(getContainers(), chePlugin.getContainers())
-        && Objects.equals(getEndpoints(), chePlugin.getEndpoints());
+    if (!super.equals(o)) {
+      return false;
+    }
+    ChePlugin plugin = (ChePlugin) o;
+    return Objects.equals(getEditors(), plugin.getEditors());
   }
 
   @Override
   public int hashCode() {
-
-    return Objects.hash(
-        super.hashCode(), getName(), getId(), getVersion(), getContainers(), getEndpoints());
+    return Objects.hash(super.hashCode(), getEditors());
   }
 
   @Override
   public String toString() {
-    return "ChePlugin{"
-        + "name='"
-        + name
-        + '\''
-        + ", id='"
-        + id
-        + '\''
-        + ", version='"
-        + version
-        + '\''
-        + ", containers="
-        + containers
-        + ", endpoints="
-        + endpoints
-        + '}';
+    return "ChePlugin{" + "editors=" + editors + "} " + super.toString();
```

Comment:
Are you sure that it's a good idea to have such output 
```
ChePlugin{editors=[....]} PluginBase{name=qwe...}
```

Reply:
I'm not sure. Do you think that it is bad?

Reply:
It may be confused in such cases:
```
LOG.debug("Debug info {} {}", chePlugin, otherPluginBase);
as output:
"Debug info ChePlugin{editors=[....]} PluginBase{name=qwe...} PluginBase{name=qwe...}"
```
Looks like three objects are printed =)
I think that it would be better to group fields of one objects, typically for Che code base with `{}`

Reply:
Makes sense